napari/_vispy/layers/points.py

```diff
@@ -192,7 +192,18 @@ def _on_shading_change(self):
         self.node.spherical = shading == 'spherical'
 
     def _on_canvas_size_limits_change(self):
-        self.node.canvas_size_limits = self.layer.canvas_size_limits
+        self.node.points_markers.canvas_size_limits = (
+            self.layer.canvas_size_limits
+        )
+        scaled_highlight = (
+            get_settings().appearance.highlight.highlight_thickness
+            * self.layer.scale_factor
+        )
+        low, high = self.layer.canvas_size_limits
+        self.node.selection_markers.canvas_size_limits = (
+            low,
```

Comment:
Don't we want the minimum size of the highlight to also be +thickness on the marker size?

Reply:
I don't know how, but I swear my brain came up with a good reason why *not* yesterday... but now I think you're right ^^'